src/pages/Calculator/Steps/SiteCondition/form.js

```diff
@@ -72,6 +45,45 @@ const SiteConditionForm = ({
   } = useSelector((s) => s.siteCondition);
 
   const [soilDrainagePrev, setSoilDrainagePrev] = useState(soilDrainage);
+  const [previousSoilDrainage, setPreviousSoilDrainage] = useState('');
```

Comment:
is there differences between these two states?

Reply:
Not at all! lol i just made my updates and copied them and moved them into this position and didn't pay attention to what was there already i guess.